python/lsst/daf/butler/configs/dimensions.yaml

```diff
@@ -142,6 +142,12 @@ elements:
     requires: [instrument]
     implies: [physical_filter, visit_system]
     metadata:
+      -
+        name: day_obs
+        type: int
+        doc: >
+          Day of observation as defined by the observatory (YYYYMMDD format).
+          If a visit crosses multiple days this entry will be null.
```

Comment:
Is this correct?  I thought it was the day when the observation is started;  LSST needs a dayObs for all data we take (not quite the same as this day_obs)

Reply:
I can change it for visit so that it returns the smallest day_obs if that's what you prefer.

Reply:
Yes please.